src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $transition-duration;
```

Comment:
please specify properties instead of using `all`
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x